custom/icds_reports/utils/__init__.py

```diff
@@ -306,7 +309,18 @@ def percent_diff(property, current_data, prev_data, all):
         tenths_of_promils = ((current_percent - prev_percent) * 10000) / (prev_percent or 1.0)
         return tenths_of_promils / 100 if (tenths_of_promils < -1 or 1 < tenths_of_promils) else 0
     else:
-        return "Data in the previous reporting period was 0"
+        return PREVIOUS_PERIOD_ZERO_DATA
+
+
+def get_color_with_green_positive(val):
```

Comment:
https://github.com/dimagi/commcare-hq/pull/22549#discussion_r237331423 applies here

Reply:
Do you mean the request for a docstring?  I included an assertion for non-numerical values as suggested.

Reply:
Yes, docstring. It's a bit nitpicky, so don't worry about it if you don't think it's important.

My reasoning: the name of this function did not clearly communicate the outcome for non-numeric inputs. However, on looking closer I see it changed since https://github.com/dimagi/commcare-hq/pull/22549#discussion_r237331423 was written, so that docstring would be wrong for this version.